impeller/compiler/compiler.cc

```diff
@@ -34,7 +34,7 @@ static CompilerBackend CreateMSLCompiler(const spirv_cross::ParsedIR& ir,
   // If this version specification changes, the GN rules that process the
   // Metal to AIR must be updated as well.
   sl_options.msl_version =
-      spirv_cross::CompilerMSL::Options::make_msl_version(1, 2);
+      spirv_cross::CompilerMSL::Options::make_msl_version(2, 1);
```

Comment:
Need to make these configurable and add a new target for it.

Reply:
Ahh right I have to fix this.

Reply:
Ok. This is now parameterized here and elsewhere.

Because it's now parameterized, the comment above no longer applies - the MSL version is set once and flows through the GN templates to here and the build_metal_library.py script.